src/bson_mapper/bson_archiver.hpp

```diff
@@ -45,6 +45,7 @@
 
 #include <chrono>
 #include <memory>
+#include <sstream>
```

Comment:
Ah I forgot we were using it to compose the key not found error, so we should just keep it.